masu/external/kafka_msg_handler.py

```diff
@@ -287,6 +287,7 @@ def process_report(report):
         LOG.info('Summarization celery uuid: %s', str(async_id))
 
 
+# pylint: disable=broad-except
```

Comment:
Is this pylint disable truly necessary? Can we narrow the scope of the except block any?

If it is necessary, please include a comment about why we need such a broad except.

Reply:
Yes that's a good idea.  I don't like the `except Exception` either.  In this case though since it can potentially block the event loop I wanted to be extra loose with the exception catching.  This block can have errors from sources like database, AWS boto APIs, Masu processing exceptions, etc.  Since I *really* don't want the service to be completely locked on any error I thought just keeping the broad `Exception` was ok here.  But I'll add a comment explaining this.